MapboxNavigation/DayStyle.swift

```diff
@@ -83,6 +83,12 @@ open class DayStyle: Style {
         EndOfRouteStaticLabel.appearance().normalTextColor = #colorLiteral(red: 0.217173934, green: 0.3645851612, blue: 0.489295125, alpha: 1)
         EndOfRouteTitleLabel.appearance().normalFont = .systemFont(ofSize: 36.0)
         EndOfRouteTitleLabel.appearance().normalTextColor = .black
+        
```

Comment:
I try to sort these entries w/o blank lines in between. If you don't have a script in Xcode you can always just copy the relevant block and do `pbpaste | sort | pbcopy` in the terminal and paste it back into Xcode.